samples/v1/test/analyzing_entities.test.yaml

```diff
@@ -0,0 +1,101 @@
+type: test/samples
```

Comment:
I see test yamlz, but I don't see any tests.  Are there generated tests to go along with and verify these?